tests/python/unittest/test_runtime_rpc.py

```diff
@@ -302,15 +318,15 @@ def check_minrpc():
         a = tvm.nd.array(np.random.uniform(size=102).astype(A.dtype), dev)
         b = tvm.nd.array(np.zeros(102, dtype=A.dtype), dev)
         time_f = f1.time_evaluator("myadd", remote.cpu(0), number=1)
-        cost = time_f(a, b).mean
+        cost = time_f(a, b).mean  # pylint: disable=unused-variable
```

Comment:
can we just rm cost =?

Reply:
yes, I wasn't sure whether to change functionality of test, although in this case probably fine to do so, I figured marking it will flag it for future test developers who are more familiar with code to decide.  This one's probably pretty safe to delete though.